assets/css/messages.css

```diff
@@ -7,6 +7,7 @@ div.messages {
 }
 
 div.status {
+  background-image: url("/build/images/message-24-ok.png");
```

Comment:
Still needs a warning icon?

Reply:
https://griidc-dev.tamucc.edu/misc/message-24-warning.png

Reply:
https://griidc-dev.tamucc.edu/misc/message-24-warning.png
Images can be taken from Drupal, so we don't have to add them to our source control.

Reply:
Added warning image